scripts/release/utils/git-client.ts

```diff
@@ -1,6 +1,18 @@
 import { simpleGit } from 'simple-git';
 
-export const git = simpleGit();
+export const git = simpleGit({
+  config: [
+    /**
+     * ensures that prereleases are listed as earlier than stable releases.
+     * eg. in the following list, this config correctly puts 7.1.0 on the top instead of the bottom:
+     * 7.1.0
+     * 7.1.0-rc.2
+     * 7.1.0-rc.1
+     * See https://stackoverflow.com/a/52680984
+     */
```

Comment:
What does it mean to be at the bottom or top of this list, please add a comment, it's not obvious what the effects are from this change.

Reply:
Updated it, is this better?